styles/styles.css

```
@@ -38,6 +38,25 @@ div[class*='-up'] .con-block.has-bg.text-block.text-list,
   margin-bottom: var(--spacing-xxs);
 }
 
+.global-navigation .feds-profile > .feds-signIn {
+  --feds-signIn-borderColor: #dadada;
+  padding: 5px 16px 6px;
+  line-height: 16.8px;
+  border: 2px solid var(--feds-signIn-borderColor);
```

It looks like the variable is only used once, so it doesn't seem necessary.
```suggestion
  padding: 5px 16px 6px;
  line-height: 16.8px;
  border: 2px solid #dadada;
```

---

Done.
Removed the variables.